style.css

```diff
@@ -0,0 +1,430 @@
+
+.fa-square-phone:hover{
+    color: #850000;
+}
+
+.content__personal:hover{
+    background: whitesmoke;
+    border: 4px solid black;
+    padding:1rem 2.5rem;
+    box-shadow: grey 0 0 0 .5em;
+}
+
+.content__skills:hover{
+    background: whitesmoke;
+    border: 4px solid black;
+    padding: 0 0 3rem 0;
+    box-shadow: grey 0 0 0 .5em;
+}
+
+.content__education:hover{
+    background: whitesmoke;
+    border: 4px solid black;
+    padding: 0 1.5em;
+    box-shadow: grey 0 0 0 .5em;
+}
+
+.content__social:hover{
+    background: whitesmoke;
+    border: 4px solid black;
+    padding: 6 .5em;
+    box-shadow: grey 0 0 0 .5em;
+}
+
+.content__header:hover{
+    background: whitesmoke;
+    border: 4px solid black;
+    padding: 2.9rem;
+    box-shadow:grey 0 0 0 .5em;
+}
+
+img{
+    max-width:17rem;
+    max-height: auto;
+    border: 1px solid black;
+    border-radius: 50%;
+    transition: .4s ease-out;   
+}
+
+img:hover{
+    max-width:20rem;
+    max-height: auto;
+    border: 1px solid black;
+}
+
+@keyframes header{
+    0%{opacity: 0; left:-30rem;}
+    100%{opacity: 1; left:0;}
+}
+    
+@keyframes personal{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+@keyframes skills{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+@keyframes education{
+    0%{opacity: 0; right:-30rem;}
+    100%{opacity: 1; right:0;}
+}
+
+
+@keyframes social{
+    0%{opacity: 0; left:-30rem;}
+    100%{opacity: 1; left:0;}
+}
+
+@media (max-width:991px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+    img:hover{
+        width: 75%;
+        max-height: auto;
+    }
+    h2{
+        font-size: 1.6em;
+    }
+    h1{
+        font-size: 2.1em;
+    }
+
+    .content__social{
+        padding: 0 .1rem;
+    }
+    .content__personal{
+        padding: 0 2rem; 
+        line-height: 150%;
+        
+    }
+    .content__personal:hover{
+        padding: .7rem 2rem; 
+        line-height: 150%;
+        word-spacing: 115%;
+        border: 2px solid;
+    }
+}
+
+@media (max-width:758px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+
+    h2{
+        font-size: 1.6rem;
+    }
+    h1{
+        font-size: 2.1rem;
+    }
+
+    .content__social{
+        padding: 0 1rem;
+    }
+
+    .links:hover{
+        border: none;
+        box-shadow: none;
+        font-size: 1em;
+        padding: .6 .5em;
+    }
+    .content__personal{
+        padding: 0 2rem; 
+        line-height: 2rem;
+
+        
+    }
+    .content__personal:hover{
+        padding: .5rem 1rem; 
+        line-height: 2rem;
+        word-spacing: 1rem;
+        border: 2px solid;
+    }
+}
+
+@media (max-width:600px){
+    img{
+        width: 65%;
+        height: auto;
+    }
+    p{
+        font-size: 1.4rem;
+    }
+    h2{
+        font-size: 1.6rem;
+    }
+    h1{
+        font-size: 2rem;
+    }
+
+    .header header-picture{
+        padding-top: 1rem;
+    }
+
+   .container{
+    display: flex;
```

Comment:
Formatting-wise, it would be good to stick to a pattern. Same amount of spaces (or tabs if you prefer), same amount of empty lines. This line misses the indentation. Perhaps you can use prettier for formatting the css file?

Reply:
Fixed.